lib/metadata/acl.js

```diff
@@ -17,12 +16,6 @@ const acl = {
         log.trace('updating object acl in metadata');
         // eslint-disable-next-line no-param-reassign
         objectMD.acl = addACLParams;
-        const replicationInfo = getReplicationInfo(objectKey, bucket, true);
```

Comment:
I'm not sure how much work it would take to make this dynamically set based on the location constraint used. If it's a small change to check the external backend type, I'd say we could put this in an `if` block so that we don't have to revisit this change should we eventually want to support this CRR operation to another Zenko instance. Also, that would support backwards compatibility with S3C releases (though maybe it's irrelevant going forward).

Reply:
Good point, I think for now I will leave as-is, as I'm not sure of the implications e.g. when mixing multiple backends where one at least would be a zenko (S3C) target. When eventually S3C will become 8.x, we will need to revisit the overall mechanism to make sure we support correctly replicating things to a S3C target amongst other clouds, and this will pop up as an action item.

As we have an integration test for ACL replication on S3C target I'm confident we will not forget to take care of this so should be fine for now especially considering the imminent release :)